File: code_wars/kyu_7/010-Most_digits.js
// ###### Solution 1
function findLongest(array) {
  let result = array[0];

  for (let i = 1; i < array.length; i++) {
    if (String(array[i]).length > String(result).length) result = array[i];
  }

  return result;
}

// ###### Solution 2
function findLongest(array) {
  return array.reduce((acc, cur) =>
    String(acc).length < String(cur).length ? cur : acc
  );
}
// ###### Link:
// ###### Problem Link: https://www.codewars.com/kata/58daa7617332e59593000006/train/javascript

File: code_wars/kyu_7/010-Most_digits.py
###### Solution 1
def find_longest(arr):
    result = arr.pop(0)

    for item in arr:
        if len(str(item)) > len(str(result)):
            result = item

    return result


###### Link:
###### Problem Link: https://www.codewars.com/kata/58daa7617332e59593000006/train/python
